js/ui/tag_box.js

```diff
@@ -736,7 +736,7 @@ const TagBox = SelectBox.inherit({
         const selectedItemsAlreadyLoaded = filteredItems.length === values.length;
         const d = new Deferred();
 
-        if(selectedItemsAlreadyLoaded) {
+        if(!this._isCacheDisabled && selectedItemsAlreadyLoaded) {
```

Comment:
Why `_isCacheDisabled`? To my mind, it's better to use `_isDataSourceChanged` or smth like this